### PR TITLE
Clean up export statements

### PR DESCRIPTION
**Pull request by Li357**
The export syntax seems a little verbose. This PR should clean up the repetition.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,87 +3,8 @@ import ReactNavigation from "react-navigation";
...
-export const DrawerItems = ReactNavigation.DrawerItems;
-export const TabView = ReactNavigation.TabView;
-export const TabBarTop = ReactNavigation.TabBarTop;
-export const TabBarBottom = ReactNavigation.TabBarBottom;
-export const SwitchView = ReactNavigation.SwitchView;
-export const withNavigation = ReactNavigation.withNavigation;
-export const withNavigationFocus = ReactNavigation.withNavigationFocus;
-export const RouterPlugin = RouterPluginDefault;
+export * from 'react-navigation'
+export { default } from './routerPlugin';
```
Comment (jspure) [inline src/index.js]: screen shot 2018-06-20 at 18 28 52

**Comment by Li357** [comment]
@jspure Thanks! I forgot to export the final export as a name:

```es6
export { default as routerPlugin } from './routerPlugin';
```

**Comment by envisean** [comment]
Just tagging @gauravprwl14 on this. Will you be able to review these PRs?

**Comment by RishabhKarnad** [comment]
Closing this PR as it is obsolete after the upgrade to `react-navigation@3` with release v0.1.0

Also, explicit exports would be preferred over exporting the entire namespace since it would give  more control over which components will be exposed by `vue-native-router`

**Pull request closed without merging**